package.json

```diff
@@ -44,6 +44,7 @@
     "file-loader": "^0.8.5",
     "gaze": "^1.0.0",
     "html-webpack-plugin": "^2.10.0",
+    "lodash": "4.6.1",
```

Comment:
Could you include it in the "dependencies" instead of "devDependencies" and change it from `"lodash": "4.6.1"` to `"lodash": "^4.6.1"`?